components/elm/src/dyn_subgrid/dyncropFileMod.F90

```diff
@@ -179,7 +179,7 @@ subroutine dyncrop_interp(bounds,crop_vars)
     allocate(pfertcft_cur(bounds%begg:bounds%endg, max_topounits, cft_lb:cft_ub))
     call pfertcft%get_current_data(pfertcft_cur)
 
-    call collapse_crop_types(wtcft_cur, nfertcft_cur, pfertcft_cur, bounds%begg, bounds%endg, verbose = .false.)
+    !call collapse_crop_types(wtcft_cur, nfertcft_cur, pfertcft_cur, bounds%begg, bounds%endg, verbose = .false.)
```

Comment:
You should add a comment about why this was commented out.

Reply:
I have reverted this edit so as to only keep BFB changes in this PR.